src/main/resources/twilio-java/creator.mustache

```diff
@@ -94,22 +99,43 @@ public class {{resourceName}}Creator extends Creator<{{resourceName}}>{
         return {{resourceName}}.fromJson(response.getStream(), client.getObjectMapper());
     }
 
-
+{{#formParams.0}}
```

Comment:
no need to conditionalize the method signature and the ending bracket (`}` line:117) separately. The whole addPostParams method block can be part of the same condition. This is applicable for addHeaderParams method as well.

Reply:
I tried making this change. But when we add both method header and closing braces in under a single condition, the method is only having one param. Other params are getting skipped